app/view/scorecard-form/scorecard-form.html

```diff
@@ -0,0 +1,874 @@
+      </text>
+      <text id="7" stroke="none" fill="none" font-family="Helvetica-Bold, Helvetica" font-size="12" font-weight="bold">
+          <tspan x="254" y="35" fill="#FFFFFF">7</tspan>
+      </text>
+      <text id="8" stroke="none" fill="none" font-family="Helvetica-Bold, Helvetica" font-size="12" font-weight="bold">
+          <tspan x="285" y="35" fill="#FFFFFF">8</tspan>
+      </text>
+      <text id="9" stroke="none" fill="none" font-family="Helvetica-Bold, Helvetica" font-size="12" font-weight="bold">
+          <tspan x="317" y="35" fill="#FFFFFF">9</tspan>
+      </text>
+      <text id="10" stroke="none" fill="none" font-family="Helvetica-Bold, Helvetica" font-size="12" font-weight="bold">
+          <tspan x="345" y="35" fill="#FFFFFF">10</tspan>
+      </text>
+      <text id="11" stroke="none" fill="none" font-family="Helvetica-Bold, Helvetica" font-size="12" font-weight="bold">
+          <tspan x="376" y="35" fill="#FFFFFF">11</tspan>
+      </text>
+      <text id="12" stroke="none" fill="none" font-family="Helvetica-Bold, Helvetica" font-size="12" font-weight="bold">
+          <tspan x="407" y="35" fill="#FFFFFF">12</tspan>
+      </text>
+      <text id="13" stroke="none" fill="none" font-family="Helvetica-Bold, Helvetica" font-size="12" font-weight="bold">
+          <tspan x="438" y="35" fill="#FFFFFF">13</tspan>
+      </text>
+      <text id="14" stroke="none" fill="none" font-family="Helvetica-Bold, Helvetica" font-size="12" font-weight="bold">
+          <tspan x="469" y="35" fill="#FFFFFF">14</tspan>
+      </text>
+      <text id="15" stroke="none" fill="none" font-family="Helvetica-Bold, Helvetica" font-size="12" font-weight="bold">
+          <tspan x="500" y="35" fill="#FFFFFF">15</tspan>
+      </text>
+      <text id="16" stroke="none" fill="none" font-family="Helvetica-Bold, Helvetica" font-size="12" font-weight="bold">
+          <tspan x="530" y="35" fill="#FFFFFF">16</tspan>
+      </text>
+      <text id="17" stroke="none" fill="none" font-family="Helvetica-Bold, Helvetica" font-size="12" font-weight="bold">
+          <tspan x="563" y="35" fill="#FFFFFF">17</tspan>
+      </text>
+      <text id="18" stroke="none" fill="none" font-family="Helvetica-Bold, Helvetica" font-size="12" font-weight="bold">
+          <tspan x="592" y="35" fill="#FFFFFF">18</tspan>
+      </text>
+      <text id="19" stroke="none" fill="none" font-family="Helvetica-Bold, Helvetica" font-size="12" font-weight="bold">
+          <tspan x="625" y="35" fill="#FFFFFF">19</tspan>
+      </text>
+      <text id="20" stroke="none" fill="none" font-family="Helvetica-Bold, Helvetica" font-size="12" font-weight="bold">
+          <tspan x="655" y="35" fill="#FFFFFF">20</tspan>
+      </text>
+      <text id="Total" stroke="none" fill="none" font-family="Helvetica-Bold, Helvetica" font-size="12" font-weight="bold">
+          <tspan x="716" y="35" fill="#FFFFFF">Total</tspan>
+      </text>
+  </svg>
```

Comment:
This file would be much easier to read if this svg was put in its own html component and then added here